src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/components/MethodologyEditableBlock.tsx

```diff
@@ -53,7 +53,7 @@ const MethodologyEditableBlock = ({
         <EditableContentBlock
           editable={editable && !isBrowser('IE')}
           id={blockId}
-          label="Block content"
+          label=""
```

Comment:
Could the label be made optional on EditableContentBlock instead of sending and empty string?